storm-core/src/clj/backtype/storm/command/dev_zookeeper.clj

```diff
@@ -14,13 +14,15 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 (ns backtype.storm.command.dev-zookeeper
-  (:use [backtype.storm zookeeper util config])
+  (:require [backtype.storm.zookeeper :as zoo]
+            [backtype.storm.config :as config])
+  (:use [backtype.storm util ])
```

Comment:
The plan is to get rid of all :use's, I had missed this one. I'll fix the trailing space here too.